test/integration/trigger.test.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+var chai = require('chai')
+  , Sequelize = require('../../index')
+  , expect = chai.expect
+  , Support = require(__dirname + '/../support')
+  , current = Support.sequelize;
+
+
+if (current.dialect.supports.tmpTableTrigger) {
+  describe(Support.getTestDialectTeaser('Model'), function() {
+    describe('trigger', function() {
+      var triggerQuery = 'create trigger User_ChangeTracking on [users] for insert,update, delete \n' +
+                          'as\n' +
+                            'SET NOCOUNT ON\n' +
+                            'if exists(select 1 from inserted)\n' +
+                            'begin\n' +
+                              'select * from inserted\n' +
+                            'end\n' +
+                            'if exists(select 1 from deleted)\n' +
+                            'begin\n' +
+                              'select * from deleted\n' +
+                            'end\n';
+      it('should return output rows after insert', function() {
+
+
+          var User = this.sequelize.define('user', {
```

Comment:
Can you move initialization of the user model and creating the trigger to a `beforeEach` function? That way we dont have to repeat the same setup for each test. 

Sorry I didn't notice this before